process/block/interceptedBlocks/common.go

```diff
@@ -166,19 +165,13 @@ func checkProof(
 		"headerHash", proof.GetHeaderHash(),
 	)
 
-	if isIncompleteProof(proof) {
+	if common.IsIncompleteProof(proof) {
```

Comment:
IsProofSizeValid cand be used here as well

Reply:
updated